db_schema.json

```diff
@@ -40,5 +41,17 @@
         ],
         "image_url": "{String}",
         "image_credit": "{String}"
+    },
+    "lendLibraryItem": {
+        "_id": "{ObjectId} ObjectId('123l4k234l')",
+        "created_at": "{String} [timestamp YYYY-MM-DD HH:mm:ss ZZ] 2013-06-13 15:30:00 -0700",
+        "updated_at": "{String} [timestamp YYYY-MM-DD HH:mm:ss ZZ] 2013-06-13 15:30:00 -0700",
+        "title": "{String}",
+        "description": "{String}",
+        "imageUrl": "{String}",
+        "userIdOwner": "{String}",
```

Comment:
Oh, I was thinking of keeping it simple (we don't track who has it now). I thought about doing the tracking, but part of the point is to build local (in person community) so the idea was to purposefully keep it low tech. The only reason I built any tech in the first place (as opposed to in person magnets / flyers) was for people to be able to search; but the goal is to get them OFF technology / screens :)

Reply:
Gotcha. In that case, should there just be an availability check?

Reply:
We could do that, though again that is more app usage & tracking that needs to be kept up to date by end users, so we'd have to have a way to track when an item was returned or not, otherwise if it's marked as "available" but isn't actually, that feature may actually be worse that not having it at all.

Reply:
I see. Okay, that makes sense!